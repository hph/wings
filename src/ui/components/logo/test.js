import React from 'react';
import renderer from 'react-test-renderer';

import Logo from './index';

describe('Logo', () => {
  it('should render an SVG logo', () => {
    expect(renderer.create(<Logo />).toJSON()).toMatchSnapshot();
  });
});
