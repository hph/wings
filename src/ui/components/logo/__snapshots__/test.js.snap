// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo should render an SVG logo 1`] = `
<div
  className="root"
  dangerouslySetInnerHTML={
    Object {
      "__html": "W.svg",
    }
  }
/>
`;
